packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -335,9 +335,9 @@ class VideoViewport extends Viewport implements IVideoViewport {
   public async play() {
     try {
       if (!this.isPlaying) {
+        this.isPlaying = true;
```

Comment:
The timing of these was causing the cine play stuff to fail.  It had originally been done after the methods returned, but that ends up not actually fixing anything, so I have move them before so the cine works.